deploy/k8s/charts/trustification/templates/workarounds/010-helm-wait-issue.yaml

```
@@ -0,0 +1,48 @@
+{{- if .Values.modules.v11yWalker.enabled }}
+{{- if .Values.workarounds.initCronJobPvcs }}
```

I think this works with having both conditions "and'd"
```suggestion
{{- if and .Values.modules.v11yWalker.enabled .Values.workarounds.initCronJobPvcs }}
```